<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>实验预约系统 - 登录</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">

    <!-- 标题 -->
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="text">
       <string>实验预约系统</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>

    <!-- 表单区域 -->
    <item>
     <layout class="QFormLayout" name="formLayout">

      <item row="0" column="0">
       <widget class="QLabel" name="labelUserId">
        <property name="text">
         <string>账号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditUserId"/>
      </item>

      <item row="1" column="0">
       <widget class="QLabel" name="labelPassword">
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>

      <item row="2" column="0">
       <widget class="QLabel" name="labelRole">
        <property name="text">
         <string>身份：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboRole"/>
      </item>

     </layout>
    </item>

    <!-- 登录按钮 -->
    <item>
     <widget class="QPushButton" name="btnLogin">
      <property name="text">
       <string>登录</string>
      </property>
      <property name="minimumHeight">
       <number>36</number>
      </property>
     </widget>
    </item>

   </layout>
  </widget>

  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>

 <resources/>
 <connections/>
</ui>
